js/components/portal-dashboard/student-navigator.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import ArrowLeftIcon from "../../../img/svg-icons/arrow-left-icon.svg";
+
+import css from "../../../css/portal-dashboard/student-navigator.less";
+import { getFormattedStudentName } from "../../util/student-utils";
+
+
+interface IProps {
+  students: any;
+  isAnonymous: boolean;
+  currentStudentIndex: any;
+  setCurrentStudent: (studentId: string | null) => void;
+  currentStudentId: string | null;
+  inResponseDetail?: boolean;
```

Comment:
I wouldn't rename props in relation to some other component.  If we refactor the component (which we've done) or have to reuse the current component elsewhere, this sort of prop naming can cause trouble.  I would call it what it is.  If you need it to change the component order, I would add a studentNameFirst prop - something that is specific to this component and doesn't require it to know anything about the outside world.  I see it gets used for a few other things, but this can probably be resolved with a better named props.  Or maybe just change it to studentNameFirst for now to at least remove the outside world dependency.